packages/chevrotain/api.d.ts

```diff
@@ -891,11 +891,21 @@ export declare class Parser {
      * @returns - The parsing rule which is the production implementation wrapped with the parsing logic that handles
      *                     Parser state / error recovery&reporting/ ...
      */
+    /* protected */ RULE(
+        name: string,
+        implementation: (...implArgs: any[]) => void,
+        config?: IRuleConfig<any>
+    ): (idxInCallingRule?: number, ...args: any[]) => CstNode
+
+    /**
+     * @see RULE
+     * @hidden
+     */
     /* protected */ RULE<T>(
         name: string,
         implementation: (...implArgs: any[]) => T,
         config?: IRuleConfig<T>
-    ): (idxInCallingRule?: number, ...args: any[]) => T | any
+    ): (idxInCallingRule?: number, ...args: any[]) => T
```

Comment:
Where is the variant the returns the CSTNode?

Reply:
@bd82 sorry, I'm not following. The variant is added just above it :)

Reply:
my bad :sad